gnovm/stdlibs/std/params.gno

```diff
@@ -0,0 +1,30 @@
+package std
+
+// These are native bindings to the banker's functions.
+func setParamString(key string, val string)
+func setParamBool(key string, val bool)
+func setParamInt64(key string, val int64)
+func setParamUint64(key string, val uint64)
+func setParamBytes(key string, val []byte)
+
+// XXX: add doc
+func SetParam(key string, val interface{}) {
```

Comment:
SetParamString etc and not expose SetParam?

Reply:
I wanted to keep the `std` package's footprint as small as possible. How much do you want to change?

Reply:
done d38c6b801